test/Sema/package_enums_treat_exhaustive.swift

```diff
@@ -302,7 +302,7 @@ public func k(_ arg: FrozenPublicEnum) -> Int {
 // package-name; requires @unknown default.
 import Utils
 
-package func n(_ arg: FrozenUfiPkgEnum) -> Int {
+package func n(_ arg: FrozenUfiPkgEnum) -> Int { // expected-error {{cannot find type 'FrozenUfiPkgEnum' in scope}}
```

Comment:
why this error? The client is in the same package and should be able to access FrozenUfiPkgEnum.

Reply:
This test is explicitly exercising `-disable-print-package-name-for-non-package-interface` so it's expected here. We should clean this up after removing `-disable-print-package-name-for-non-package-interface`.